File: roles/dev/files/cplusplus/test/util/util_test.cc
#include <gtest/gtest.h>

#include <vector>
#include <string>

#include "util/util.h"

TEST(UtilTest, JoinStringsConcatenatesWithCommaSeparator) {
  const auto inputs = std::vector<std::string_view>{"one", "two"};
  EXPECT_EQ(Util::JoinStrings(inputs), "one, two");
}

File: roles/ja-apex-dev/jg-go/files/go/util/util_test.go
package util_test

import (
	"testing"
	"github.com/stretchr/testify/assert"
	"example.com/go-template/util"
)

func TestJoinStringsSeparator(t *testing.T) {
	assert.Equal(t, "One, Two", util.JoinStrings("One", "Two"))
}
